plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -19,6 +19,7 @@
 #include <fc/time.hpp>
 
 #include <boost/asio.hpp>
+#include <boost/asio/deadline_timer.hpp>
```

Comment:
deadline_timer has been deprecated since 1.87, and in 1.89 it has been removed from the `<boost/asio.hpp>` include. We should spend a few minutes some time and refactor to system_timer (I don't think too difficult).

Reply:
okay I made #1830 but I don't think we'll forget forever since eventually deadline_timer will just be removed :grinning: